<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyBox</class>
 <widget class="QWidget" name="PropertyBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>788</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Box properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>324</width>
        <height>786</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="11" column="0" colspan="3">
        <widget class="QPushButton" name="boxExpand">
         <property name="toolTip">
          <string>It's replacing the box by a grid</string>
         </property>
         <property name="text">
          <string>Expand to Grid</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="OrientationLabel">
         <property name="text">
          <string>Orientation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="GenericPropertiesHead" name="genericPropsHeadWidget" native="true"/>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QDoubleEdit" name="volumeLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SizeLayout_3">
         <item>
          <layout class="QVBoxLayout" name="XLayout_3">
           <item>
            <widget class="QLabel" name="XCenterLabel_2">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="XOrientationInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="YLayout_3">
           <item>
            <widget class="QLabel" name="YCenterLabel_2">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="YOrientationInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ZLayout_3">
           <item>
            <widget class="QLabel" name="ZCenterLabel_2">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="ZOrientationInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="BoxCenterLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Box center</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="SizeLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="3">
        <widget class="SubPropertyClipping" name="subPropertyClipping" native="true"/>
       </item>
       <item row="3" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SizeLayout_2">
         <item>
          <layout class="QVBoxLayout" name="XLayout_2">
           <item>
            <widget class="QLabel" name="XCenterLabel">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="XCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="YLayout_2">
           <item>
            <widget class="QLabel" name="YCenterLabel">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="YCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ZLayout_2">
           <item>
            <widget class="QLabel" name="ZCenterLabel">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="ZCenterInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="12" column="0" colspan="3">
        <widget class="QGroupBox" name="gridBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QRadioButton" name="multipleModeRadioBtn">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Multiple</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="SizeLayout_5">
            <item>
             <layout class="QVBoxLayout" name="XLayout_5">
              <item>
               <widget class="QLabel" name="XGridLabel">
                <property name="text">
                 <string>X</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleEdit" name="XGridInfield">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="YLayout_5">
              <item>
               <widget class="QLabel" name="YGridLabe">
                <property name="text">
                 <string>Y</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleEdit" name="YGridInfield">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="ZLayout_5">
              <item>
               <widget class="QLabel" name="ZGridLabel">
                <property name="text">
                 <string>Z</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleEdit" name="ZGridInfield">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDivision">
            <property name="text">
             <string>Division By : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="stepModeRadioBtn">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>Step</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="14" column="0" rowspan="2" colspan="3">
        <widget class="GenericPropertiesFeet" name="genericPropsFeetWidget" native="true"/>
       </item>
       <item row="7" column="0" colspan="3">
        <layout class="QHBoxLayout" name="SizeLayout_4">
         <item>
          <layout class="QVBoxLayout" name="XLayout_4">
           <item>
            <widget class="QLabel" name="XSizeLabel">
             <property name="text">
              <string>X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="XSizeInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="YLayout_4">
           <item>
            <widget class="QLabel" name="YSizeLabel">
             <property name="text">
              <string>Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="YSizeInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ZLayout_4">
           <item>
            <widget class="QLabel" name="ZSizeLabel">
             <property name="text">
              <string>Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleEdit" name="ZSizeInfield">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>0</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="16" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GenericPropertiesHead</class>
   <extends>QWidget</extends>
   <header location="global">GenericPropertiesHead.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GenericPropertiesFeet</class>
   <extends>QWidget</extends>
   <header>GenericPropertiesFeet.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qdoubleedit.h</header>
  </customwidget>
  <customwidget>
   <class>SubPropertyClipping</class>
   <extends>QWidget</extends>
   <header>SubPropertyClipping.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>XCenterInfield</tabstop>
  <tabstop>YCenterInfield</tabstop>
  <tabstop>ZCenterInfield</tabstop>
  <tabstop>XOrientationInfield</tabstop>
  <tabstop>YOrientationInfield</tabstop>
  <tabstop>ZOrientationInfield</tabstop>
  <tabstop>XSizeInfield</tabstop>
  <tabstop>YSizeInfield</tabstop>
  <tabstop>ZSizeInfield</tabstop>
  <tabstop>stepModeRadioBtn</tabstop>
  <tabstop>multipleModeRadioBtn</tabstop>
  <tabstop>XGridInfield</tabstop>
  <tabstop>YGridInfield</tabstop>
  <tabstop>ZGridInfield</tabstop>
 </tabstops>
 <resources>
  <include location="../../../openscantools.qrc"/>
 </resources>
 <connections/>
</ui>
